workflow-runtime/src/commonTest/kotlin/com/squareup/workflow1/RenderWorkflowInTest.kt

```diff
@@ -75,14 +109,28 @@ class RenderWorkflowInTest {
     }
 
     testScope.cancel()
-    renderWorkflowIn(workflow, testScope, MutableStateFlow(Unit)) {}
+    renderWorkflowIn(
+      workflow,
+      testScope,
+      MutableStateFlow(Unit),
+      runtimeConfig = RuntimeConfig(processMultipleActions)
+    ) {}
     testScope.advanceUntilIdle()
 
     assertFalse(sideEffectWasRan)
   }
 
   @Test
-  fun `side effects from initial rendering in non-root workflow are never started when scope cancelled before start`() { // ktlint-disable max-line-length
+  fun `side effects from initial rendering in non-root workflow are never started when scope cancelled before start baseline`() =
```

Comment:
Seems like this would be a good use of a parameterized test / `@Parameters`. (Fine to wait for follow up, if ever.)

Reply:
- [ ] Use Junit @ParametrizedTest for the runtime tests.

Reply:
Welp, couldn't use JUnit because in common code - but made our own